packages/contractkit/src/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './kit'
 export { GenesisBlockUtils } from './network-utils/genesis-block-utils'
 export { StaticNodeUtils } from './network-utils/static-node-utils'
 export { CeloProvider } from './providers/celo-provider'
+export * from './utils/phone-number-lookup/phone-number-lookup-utils'
```

Comment:
Sorry, I should have been more clear. This approach still exports everything from under /phone-number-lookup/ into the global namespace. What I was suggesting was wrapping the exports into an object. Something like:
```
import foo from bar

const PhoneNumberUtils = {
  foo,
}
```

Then export `PhoneNumberUtils` here